16/alpine3.17/Dockerfile

```diff
@@ -53,6 +55,8 @@ RUN addgroup -g 1000 node \
     && curl -fsSLO --compressed "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION.tar.xz" \
     && curl -fsSLO --compressed "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc" \
     && gpg --batch --decrypt --output SHASUMS256.txt SHASUMS256.txt.asc \
+    && gpgconf --kill all \
+    && rm -rf "$GNUPGHOME" \
```

Comment:
Minor, but maybe this should move with the rest of the cleanups at line 69

Reply:
Yeah I wasn't sure about the best location and chose to follow the PHP-approach; this includes a manual `gpgconf --kill all` which I'm not sure is necessary (i.e. it works fine on my end without it, but maybe it has a more specific purpose) which can only be run when `gpg` is still installed. So in that case, I could only move it down to be just above the `apk del` call. If you prefer that, let me know and I'll update it!

Reply:
@nschonni as mentioned, if we keep the `gpgconf --kill all` call, this can only be moved as far down as just above line 66. Do you still prefer this, or shall we keep it the current way.

Alternatively, we could just merge this for now, as it is functionally equivalent, move the discussion to #1901 and find a definitive approach there.